### DISCUSSION TITLE
How do I set the time period for Wino Mail to load emails to local storage?

### DISCUSSION THREAD

**Discussion by sao321** [bkaankose/Wino-Mail#278] [Q&A]
Hi,

How can I set the time period for Wino Mail to load emails to the local disk, for example, to only load the last 3 months of emails?

Thanks

**Comment by Tiktack** [comment]
Why would you want that? My two mailboxes with I believe around 40K emails in total taking 25MB of disk space. I think disk space shouldn't be a problem. Any other reason?

**Comment by RenaudS2** [comment]
Useless answer...

My answer : Verrrrrry looonngg......
I have a mailbox from 2007.... a lot of mail... it took several minutes to load ALL the email from 2007 each time a refresh the inbox....
It's just unusable.

**Comment by RenaudS2** [comment]
If a solution exists to load only 3 last month, and not each time i go to my inbox, please explain.
Thx

**Comment by bkaankose** [comment]
Wino Mail v2 implemented this functionality. You can now select a date in the past to start with or sync everything as you like.

Closing as fixed.
